src/main/features/response/form/models/statement-of-means/maintenance.ts

```diff
@@ -3,17 +3,17 @@ import { IsDefined, IsInt, Min, ValidateIf } from 'class-validator'
 import { Serializable } from 'models/serializable'
 import { toNumberOrUndefined } from 'common/utils/numericUtils'
 import * as toBoolean from 'to-boolean'
-import { ValidationErrors } from 'forms/validation/validationErrors'
+import { ValidationErrors as GlobalValidationErrors } from 'forms/validation/validationErrors'
 
 export class Maintenance implements Serializable<Maintenance> {
 
-  @IsDefined({ message: ValidationErrors.YES_NO_REQUIRED })
+  @IsDefined({ message: GlobalValidationErrors.YES_NO_REQUIRED })
   option: boolean
 
   @ValidateIf(o => o.option === true)
-  @IsDefined({ message: ValidationErrors.NUMBER_REQUIRED })
-  @IsInt({ message: ValidationErrors.NUMBER_REQUIRED })
-  @Min(1, { message: ValidationErrors.NUMBER_REQUIRED })
+  @IsDefined({ message: GlobalValidationErrors.NUMBER_REQUIRED })
+  @IsInt({ message: GlobalValidationErrors.INTEGER_REQUIRED })
+  @Min(1, { message: GlobalValidationErrors.POSITIVE_NUMBER_REQUIRED })
```

Comment:
We usually use `@IsPositive` in such a case which is probably more expressive. Any reason not to use it?